### ISSUE TITLE
using 'article' editor & multiple locale records leads to data loss 

### ISSUE THREAD

**Issue by 0spinboson** [bolt/core#1746]
rc35, symfony 5.13, php 7.2, article version from yesterday
When I set article as the editor for a field that has localize=true, and I edit the field in another locale, I've repeatedly run into data being lost. So e.g. I was editing the dutch locale, and after saving, the english locale of that same field no longer contains any content.

**Comment by I-Valchev** [comment]
Can't reproduce yet.

Let's look at this together @bobdenotter .

**Comment by bobdenotter** [comment]
@0spinboson On second thought, we can't seem to reproduce this yet.. Are you certain data is actually lost?

**Comment by 0spinboson** [comment]
It happened about 5x, with me editing the field content in english then saving, only to find wiped content field in the dutch locale, and vice versa. I don't really know how to supply additional info, though. But after switching back to the redactor editor, it has never happened yet/again.

**Comment by I-Valchev** [comment]
Let's see if this shows up agian. In the meantime, I'll bump that to 4.1

**Comment by denis-gorin** [comment]
@0spinboson, please tell, have you changed `localize` attribute in `config\bolt\contenttypes.yaml` between savings of an entry?
I suppose I could to reproduce this, but not really sure. my way was: 
- create entry fields structure (ex. `collection` of `set`)
- fill with content
- add `localize` attribute to the one of a set's field
- refill content depending on locale - cause this field data loss in a different locale

**Comment by 0spinboson** [comment]
no, I just changed it once, it usually bugged at least the first time, though on one of my sites it did so pretty consistently.
